include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -31,117 +31,123 @@
 // DAMAGE.
 //
 // ==========================================================================
-// Author: Hannes Hauswedell <hannes.hauswedell@fu-berlin.de>
-// Author: Marcel Ehrhardt <marcel.ehrhardt@fu-berlin.de>
-// Author: David Heller <david.heller@fu-berlin.de>
-// ==========================================================================
+
+/*!\file
```

Comment:
> it will produce the same result as using \file seqan3/alphabet/union_alphabet.hpp or \file alphabet/union_alphabet.hpp or \file union_alphabet.hpp.

These three are not equivalent, that's the point :)

If it produces documentation like the second (and likely the first), we could switch to it. If it produces documentation like the last, we cannot, because there will be conflicts between files of the same name.